packages/world/src/interfaces/IWorldCold.sol

```diff
@@ -0,0 +1,48 @@
+// SPDX-License-Identifier: MIT
```

Comment:
Why do we need to explicitly create this interface instead of using `worldgen` with the system implementing the functions? (like we do with `IRegistrationSystem`)

Reply:
because of `IWorldCold_grantAccessWithName`, it's either hardcode a selector, or define the 2 interfaces manually. Manual is more work, but no risk of divergence between the hardcoded method selector and its true version (because the system inherits the manual interface)